### PR TITLE
MM-20838 Add unit tests to the "ConfigShowCmd" mmctl command

### PR DESCRIPTION
**Pull request by nadalfederer**
Summary:
Add unit tests to the "PluginListCmd" mmctl command

JIRA: https://mattermost.atlassian.net/browse/MM-20838
closes https://github.com/mattermost/mattermost-server/issues/13286

### CODE REVIEW

File: commands/config_test.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestConfigShowCmd() {
+	s.Run("Should show config", func() {
+		printer.Clean()
+		configShowArg := "config-show"
```
Comment (mgdelacroix) [inline commands/config_test.go]: The config show command expects no args

File: commands/config_test.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			Return(mockConfig, &model.Response{Error: nil}).
+			Times(1)
+
+		err := configShowCmdF(s.client, &cobra.Command{}, []string{configShowArg})
+		s.Require().Nil(err)
+		s.Len(printer.GetLines(), 1)
+		s.Equal(mockConfig, printer.GetLines()[0])
+		s.Len(printer.GetErrorLines(), 0)
+	})
+
```
Comment (mgdelacroix) [inline commands/config_test.go]: nit: unnecessary whitespace

**Comment by nadalfederer** [comment]
@mgdelacroix thanks for the review. I fixed current issues

**Comment by nadalfederer** [comment]
yeah, my bad, fix it

**Comment by jasonblais** [comment]
Re-requested reviews for after the holidays, thanks! :)

**Comment by jasonblais** [comment]
@mgdelacroix quick reminder to help with reviews when you have a chance, thanks! :)
